src/components/movie-details/movie-details.tsx

```diff
@@ -0,0 +1,39 @@
+import styles from './styles.module.scss'
+
+interface Props {
+  imgURL: string
+  name: string
+  genres: string[]
+  year: number
+  rating: number
+  duration: string
+  description: string
+}
+
+export const MovieDetails = ({
+  imgURL,
+  name,
+  year,
+  rating,
+  duration,
+  description,
+  genres,
+}: Props) => (
+  <div className={styles.container}>
+    <div>
+      <img alt={name} className={styles.poster} src={imgURL} />
```

Comment:
Not react, but good to know, I think, that it will be better to use `""` as `alt` here:

```suggestion
      <img alt="" className={styles.poster} src={imgURL} />
```